exercises/practice/resistor-color/tests.ua

```diff
@@ -0,0 +1,13 @@
+~ "resistor-color.ua" ~ ColorCode Colors
+
+# Color Black
+⍤⤙≍ 0 ColorCode {"black"}
+
+# Color White
+⍤⤙≍ 9 ColorCode {"white"}
+
+# Color Orange
+⍤⤙≍ 3 ColorCode {"orange"}
```

Comment:
Could you make it such that `ColorCode` can be called with a regular, unboxed string? As in: have `ColorCode` do the boxing.